### ISSUE TITLE
endpoint to get a .png badge/shield with the generation/co2 of a zone

### ISSUE THREAD

**Issue by Kezii** [electricitymaps/electricitymaps-contrib#4932]
feature request: an endpoint in the app that returns a .png with the details of a zone, like this

![immagine](https://user-images.githubusercontent.com/3357750/211121748-80c50787-04d7-44e8-9a76-ac5134ff28ce.png)

it should return a .png in the same style of the app, with the same data, etc
to integrate it in another website / bot etc

**Comment by VIKTORVAV99** [comment]
I don't think we will ever create a api endpoint that returns a static image, it would be lacking too much information like data sources, carbon intensities, etc.

On the other hand creating a interactive widget could be cool but I'll be honest and say it's low in the list of priorities at the moment.

Since this is not really an issue I will be converting this to a discussion, I hope you understand.
